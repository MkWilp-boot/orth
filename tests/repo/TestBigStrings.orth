@define SOME_STR s "Hello, I am a reaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaallllllllllllllly big string :)\n"
@define NORMAL s "Normal size string\n"

proc main with 0 out 0 in
    SOME_STR puts
    NORMAL puts
end