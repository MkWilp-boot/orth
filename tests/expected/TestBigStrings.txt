Hello, I am a reaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaallllllllllllllly big string :)
Normal size string
